src/main/java/seedu/address/logic/commands/LinkCommand.java

```diff
@@ -84,12 +84,12 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(String.format(
                     MESSAGE_LINKED_PERSON,
```

Comment:
Do you think we should change line 36 to `MESSAGE_LINKED_PERSON = "%1$s is already linked to %2$s"` instead?